### PR TITLE
feat(console): add assigned users column to roles table

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

feat(console): add assigned users column to roles table

## Testing

image

### CODE REVIEW

File: packages/console/src/pages/Roles/components/AssignedUsers/index.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <div className={styles.users}>
+      <div className={styles.avatars}>
+        {users.map(({ id, avatar }) => (
+          <UserAvatar key={id} url={avatar} className={styles.avatar} />
+        ))}
+      </div>
+      <span className={styles.count}>{count.toLocaleString()}</span>
+    </div>
+  ) : (
+    <span className={styles.empty}>-</span>
```
Comment (simeng-li) [inline packages/console/src/pages/Roles/components/AssignedUsers/index.tsx:23]: nit: better provide a consistent type of box model in the component.  div is a block model item, but the span is an inline model item.

**Review by simeng-li — COMMENTED** [review]
overall LGTM

File: packages/console/src/pages/Roles/components/AssignedUsers/index.tsx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/pages/Roles/components/AssignedUsers/index.tsx]: updated.

File: packages/console/src/pages/Roles/components/AssignedUsers/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+const AssignedUsers = ({ users, count }: Props) =>
+  count > 0 ? (
+    <div className={styles.users}>
+      <div className={styles.avatars}>
+        {users.map(({ id, avatar }) => (
+          <UserAvatar key={id} url={avatar} className={styles.avatar} />
+        ))}
+      </div>
+      <span className={styles.count}>{count.toLocaleString()}</span>
```
[the remark this answers is not in the document]
Comment (xiaoyijun) [reply packages/console/src/pages/Roles/components/AssignedUsers/index.tsx]: Updated with the new design:
image

**Review by simeng-li — APPROVED** [review]

**Pull request merged**